### ISSUE TITLE
What's the name of the app?

### ISSUE THREAD

**Issue by odonckers** [lmmenge/WatchSpringboard-Prototype#11]
This looks really cool, but I would like to try it on my iPad. Could you tell us the name of the app?

**Comment by shirerom** [comment]
Of course, it's called **HeyEcho**. But it is made and optimized for iPhone.

A short explanation: I developed HeyEcho to share moments with my friends and family and NOT with the whole world. So, if you expect to receive content from stranger, this is not your app. 

If you want to use HeyEcho in a meaningful way, then you need at least one buddy, which also has installed HeyEcho and you have each other in the address book.

Have fun! :)
